### PR TITLE
Add json for Varanidae keyboard

### PR DESCRIPTION
**Pull request by TokageItLab**
## Description

Add json for Varanidae keyboard recently merged in qmk_firmware.

## QMK Pull Request 

- https://github.com/qmk/qmk_firmware/pull/21340

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/varanidae/varanidae.json [hunk 1]
```diff
@@ -0,0 +1,296 @@
+{
+  "name": "Varanidae",
+  "vendorId": "0x5369",
+  "productId": "0x0001",
+  "keycodes": [],
+  "menus": [],
```
Comment (Xelus22) [inline v3/varanidae/varanidae.json:6]:
```suggestion
```

Remove unused json keys.

**Review by Xelus22 — CHANGES_REQUESTED** [review]
Please remove v2 definition (src/ folder version) as that is for legacy support of boards

File: v3/varanidae/varanidae.json (hunk 1 above)
Comment (TokageItLab) [reply v3/varanidae/varanidae.json]: Done, thanks.

**Comment by TokageItLab** [comment]
Removede `src/varanidae`

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**